README.md

```diff
@@ -1,30 +1,145 @@
 # rav1d
 
-**rav1d** is an **AV1** cross-platform **d**ecoder, open-source, and focused on speed and correctness. It is a Rust port of [dav1d](https://code.videolan.org/videolan/dav1d).
+**rav1d** is an AV1 cross-platform decoder, open-source, and focused on speed
+and correctness. It is a Rust port of
+[dav1d](https://code.videolan.org/videolan/dav1d).
 
-rav1d is currently experimental. Core functionality has been transpiled using [c2rust](https://github.com/immunant/c2rust), but not everything has been ported yet and the transpiled code needs to be cleaned up from unsafe transpiled Rust to safe, idiomatic Rust.
+# Building
+
+rav1d is written in Rust and uses the standard Rust toolchain to build. The Rust
+toolchain can be installed by going to https://rustup.rs. The rav1d library
+builds on stable Rust for x86, x86_64, and aarch64, but currently requires a
+nightly compiler for armv7 and riscv. The project is configured to use a nightly
+compiler by default via `rust-toolchain.toml`, but a stable build can be made
+with the `+stable` cargo flag.
```

Comment:
You also need to only build the library with `--lib`.  By default `cargo build` will try to build the other binaries, too, which still use a bunch of nightly features.

Reply:
Oh I see you put the `--lib` below.  Maybe just add here "a stable `--lib` build" or "a stable build of the library".